### PR TITLE
Add support for backing up base type storage options

### PR DESCRIPTION
**Pull request by chrishajas**

### CODE REVIEW

File: backup/queries_relations.go [hunk 1]
```diff
@@ -354,7 +354,7 @@ func GetPartitionTemplates(connection *utils.DBConn) map[uint32]string {
 }
 
 func GetStorageOptions(connection *utils.DBConn) map[uint32]string {
```
Comment (khuddlefish) [inline backup/queries_relations.go]: Is it confusing that we are calling both table storage options and base type storage options "StorageOptions"?

Comment (chrishajas) [reply backup/queries_relations.go]: They do represent the same thing. For columns I think we call them encoding, but they all are "storage options", just for different objects.

Comment (khuddlefish) [reply backup/queries_relations.go]: Sure, but I meant maybe we should qualify it with "TypeStorageOptions" and "TableStorageOptions" or something. They are essentially the same thing but on different objects.

Comment (chrishajas) [reply backup/queries_relations.go]: Just the global function or the struct fields too?

Comment (khuddlefish) [reply backup/queries_relations.go]: The struct fields are probably ok since they're qualified by being a part of the type object. Mostly the function.

**Pull request merged**